brainzutils/musicbrainz_db/utils.py

```diff
@@ -76,6 +105,7 @@ def get_entities_by_ids(query, entity_type, ids):
         query (Query): SQLAlchemy Query object.
         entity_type (str): Type of entity being queried.
         ids (list): IDs of the target entities.
+        unknown_entities_for_missing (bool): True, if NoDataFoundException should be suppressed and unknown entities should be returned instead.
```

Comment:
If set, `NoDataFoundException` is suppressed and unknown entities are returned instead. (docstrings are guided to have present tense).

Reply:
Updated.